### PR TITLE
Fix `Slider` rendering and align React docs and demo with the `web` version

### PR DESCRIPTION
**Pull request by adamkudrna**
## Description

obrazek

### Additional context

- Fixed rendering in Chromium browsers
- Introduce `disabled` state
- Remove `require` and `selected` states
- Improve README of React implementation

### Issue reference

https://jira.almacareer.tech/browse/DS-1325

### CODE REVIEW

**Review by literat — APPROVED** [review]
Nice, thanks 👍

File: packages/web/src/scss/components/UNSTABLE_Slider/index.html [hunk 1]
```diff
@@ -21,13 +21,12 @@ <h2 class="docs-Heading">Default</h2>
...
         type="range"
-        min="0"
-        max="10"
-        step="1"
-        value="3"
-        oninput="this.style.setProperty('--slider-position', `${Math.round((100 * this.value) / 10)}%`);"
+        min="3"
+        max="12"
+        value="9"
+        oninput="this.style.setProperty('--slider-position', `${Math.round(100 * (this.value - 3) / (12 - 3))}%`);"
```
Comment (dlouhak) [inline packages/web/src/scss/components/UNSTABLE_Slider/index.html:29]: Name `Custom Steps` evoked in me that the demonstration will include settings of `max`, `min`, `step` and will display the currently selected value (so a small playground like in the case of Toast, etc.), but I could imagine that the time wasn't on your side and we can extend this demo later.

Comment (dlouhak) [reply packages/web/src/scss/components/UNSTABLE_Slider/index.html:29]: Btw, couldn't this be part of CSS?

```
.UNSTABLE_Slider__input {
    --slider-position: math.round(100 * (attr(value) - attr(min)) / (attr(max) - (attr(min))));`
    …
}
```
?

File: packages/web/src/scss/components/UNSTABLE_Slider/README.md [hunk 2]
```diff
@@ -53,24 +55,10 @@ property needs to be present when the Slider is initially rendered (see the `sty
 value changes (see the `oninput` handler with the calculation) which includes also the user's interaction with the
```
Comment (dlouhak) [inline packages/web/src/scss/components/UNSTABLE_Slider/README.md:55]:
```suggestion
value changes (see the `oninput` handler with the calculation above) which includes also the user's interaction with the
```

File: packages/web-react/src/components/UNSTABLE_Slider/stories/UNSTABLE_Slider.stories.tsx [hunk 3]
```diff
@@ -115,8 +108,7 @@ const meta: Meta<typeof UNSTABLE_Slider> = {
     id: 'slider',
     isFluid: false,
     isLabelHidden: false,
-    isRequired: false,
-    isSelected: false,
+    isDisabled: false,
```
Comment (dlouhak) [inline packages/web-react/src/components/UNSTABLE_Slider/stories/UNSTABLE_Slider.stories.tsx:111]: 🔤 ?

File: packages/web-react/src/components/UNSTABLE_Slider/stories/UNSTABLE_Slider.stories.tsx [hunk 4]
```diff
@@ -45,16 +45,9 @@ const meta: Meta<typeof UNSTABLE_Slider> = {
         defaultValue: { summary: 'false' },
       },
     },
-    isRequired: {
+    isDisabled: {
```
Comment (dlouhak) [inline packages/web-react/src/components/UNSTABLE_Slider/stories/UNSTABLE_Slider.stories.tsx:48]: 🔤 ?

File: packages/web-react/src/components/UNSTABLE_Slider/README.md [hunk 5]
```diff
@@ -72,6 +151,9 @@ On top of the API options, the components accept [additional attributes][readme-
 If you need more control over the styling of a component, you can use [style props][readme-style-props]
 and [escape hatches][readme-escape-hatches].
 
+[mdn-range]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range
+[html-spec-range]: https://html.spec.whatwg.org/multipage/input.html#range-state-(type=range)
 [readme-additional-attributes]: https://github.com/lmc-eu/spirit-design-system/blob/main/packages/web-react/README.md#additional-attributes
 [readme-escape-hatches]: https://github.com/lmc-eu/spirit-design-system/blob/main/packages/web-react/README.md#escape-hatches
 [readme-style-props]: https://github.com/lmc-eu/spirit-design-system/blob/main/packages/web-react/README.md#style-props
+[dictionary-validation]: https://github.com/lmc-eu/spirit-design-system/blob/main/docs/DICTIONARIES.md#validation
```
Comment (crishpeen) [inline packages/web-react/src/components/UNSTABLE_Slider/README.md:159]: abc please

**Review by crishpeen — APPROVED** [review]

**Review by dlouhak — APPROVED** [review]

**Pull request merged**